src/pages/signin/ResendValidationForm.js

```diff
@@ -34,6 +34,9 @@ const propTypes = {
         closed: PropTypes.bool,
     }),
 
+    /** Title to be shown in the form */
+    titleMessage: PropTypes.string.isRequired,
```

Comment:
There are a couple of issues with this:

1.  We are not passing a _message_ here we are passing a _translation key_
2. It would be better to pass along something like an `errors` object and then have the form itself define the translation keys to use.